### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by tsy24**
Increase font size in sequence diagrams
Add manual testing for tag related commands and undo/redo

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1028,7 +1036,56 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Prerequisites: List all elderlies using the `viewElderly` command. Multiple elderlies in the list. The first elderly in the list has one `diabetes` tag.
+
+    2. Test case: `addTag 1 t/hypertension`<br>
+       Expected: `hypertension` tag is added to the first elderly in the list.
+   
+    3. Test case: `addTag 1 t/vegetarian t/flu`<br>
+       Expected: `vegetarian` and `flu` tags are added to the first elderly in the list. 
+   
+    4. Invalid commands to try (Error details shown in the status message):
```
Comment (nicole-luo-exe) [inline docs/DeveloperGuide.md:1052]: Could also include the multiple tags with same/different casing

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1028,7 +1036,56 @@ testers are expected to do more *exploratory* testing.
 
    4. Invalid commands to try (Error details shown in the status message):
       * Invalid index >= size of elderly list or <= 0: `deleteElderly 5` or `deleteElderly -1`
+      
+### Adding tags to an elderly
 
+1. Add one or more tags to an elderly in NurseyBook
+
+    1. Prerequisites: List all elderlies using the `viewElderly` command. Multiple elderlies in the list. The first elderly in the list has one `diabetes` tag.
```
Comment (nicole-luo-exe) [inline docs/DeveloperGuide.md:1044]: Does it have to be all? `findElderly` and `filter` should be okay as well.

Comment (nicole-luo-exe) [reply docs/DeveloperGuide.md:1044]: Ohh I see, it's just for this particular test case's scenario?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1125,24 +1182,56 @@ testers are expected to do more *exploratory* testing.
...
-   
+
     5. Test case - a week ahead:`viewSchedule 2021-11-19` <br>
        Expected: Task B should be displayed. Date of Task B in this display should be `2021-11-19`.
-   
-    6. Some other incorrect `viewSchedule` commands to try (error details shown in status message): 
+
+    6. Invalid commands to try (Error details shown in the status message):
         * Date input has already passed: `viewSchedule 2021-11-11`
         * Not formatting the date correctly in yyyy-mm-dd format: `viewSchedule 16-12-2021`
```
Comment (nicole-luo-exe) [inline docs/DeveloperGuide.md:1200]: Maybe also dates that go beyond the acceptable 12 weeks?

File: docs/diagrams/FilterSequenceDiagramPart2.puml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+[-> LogicManager
+activate LogicManager
+
+LogicManager -> FilterCommand : execute()
+activate FilterCommand
+
+FilterCommand -> Model : updateFilteredElderlyList(p)
+activate Model
+
+loop for every elderly
```
Comment (nicole-luo-exe) [inline docs/diagrams/FilterSequenceDiagramPart2.puml:23]: "not all elderlies tested" is better? Not too sure though my UML abilities are....

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1125,24 +1182,56 @@ testers are expected to do more *exploratory* testing.
...
+
+    1. Prerequisites: NurseyBook is just launched with no commands entered yet.
+
+    2. Test case: enter `addElderly en/Khong Guan a/80 g/M r/10` then `undo` and then `redo`<br>
+       Expected: Addition of elderly `Khong Guan` is executed again. List of elderly displayed contains `Khong Guan`.
+
+    3. Test case: `redo`<br>
+       Expected: No change in data of NurseyBook. Error details shown in the status message.
+
+    4. Test case: enter `addElderly en/Khong Guan a/80 g/M r/10` and then `redo`<br>
```
Comment (nicole-luo-exe) [inline docs/DeveloperGuide.md:1232]: Is this addElderly supposed to be an erroneous input?

**Review by nicole-luo-exe — CHANGES_REQUESTED** [review]
Little tweaks, but overall good job!

File: docs/diagrams/FilterSequenceDiagramPart2.puml (hunk 4 above)
Comment (nicole-luo-exe) [reply docs/diagrams/FilterSequenceDiagramPart2.puml:23]: This is in reference to the loop [condition]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tsy24) [reply docs/DeveloperGuide.md:1052]: Ok will add it in!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (tsy24) [reply docs/DeveloperGuide.md:1044]: Yep this part is just some suggestions for manual testing. So this is a scenario.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (tsy24) [reply docs/DeveloperGuide.md:1200]: Ok will add it in!

**Review by nicole-luo-exe — APPROVED** [review]
LGTM!

**Pull request merged**